### PR TITLE
Update dead links

### PR DESCRIPTION
**Pull request by ganesshkumar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1771,9 +1770,7 @@ Handy metrics based on numbers above:
 * [Microsoft Engineering](https://engineering.microsoft.com/)
 * [Microsoft Python Engineering](https://blogs.msdn.microsoft.com/pythonengineering/)
 * [Netflix Tech Blog](http://techblog.netflix.com/)
-* [Paypal Developer Blog](https://devblog.paypal.com/category/engineering/)
 * [Pinterest Engineering Blog](https://medium.com/@Pinterest_Engineering)
-* [Quora Engineering](https://engineering.quora.com/)
```
Comment (duynguyenhoang) [inline README.md]: Could you please use this link `https://www.quora.com/q/quoraengineering`, instead of remove it

File: README.md [hunk 2]
```diff
@@ -1771,9 +1770,7 @@ Handy metrics based on numbers above:
 * [Microsoft Engineering](https://engineering.microsoft.com/)
 * [Microsoft Python Engineering](https://blogs.msdn.microsoft.com/pythonengineering/)
 * [Netflix Tech Blog](http://techblog.netflix.com/)
-* [Paypal Developer Blog](https://devblog.paypal.com/category/engineering/)
```
Comment (duynguyenhoang) [inline README.md]: Paypal moved their blog to medium https://medium.com/paypal-engineering, please update to this new link

Comment (ganesshkumar) [reply README.md]: Updated

File: README.md (hunk 1 above)
Comment (ganesshkumar) [reply README.md]: Updated

**Comment by ganesshkumar** [comment]
@duynguyenhoang I have resolved your comments

**Comment by donnemartin** [comment]
@ganesshkumar @duynguyenhoang @murugeshkumark thank you!

**Pull request merged**